src/result_collection.jl

```diff
@@ -84,6 +86,8 @@ function collect_results!(filename, folder;
     verbose = true,
     update = false,
     newfile = false, # keyword only for defining collect_results without !
+    rinclude = [r""],
+    rexclude = [r"^\b$"],
```

Comment:
What do the default regex expressions mean? I thought by default we excluded nothing...?

Reply:
That's what those 2 expressions do - Regex can be surprisingly convoluted.

On that note though, I will skip the filtering if the Regex matches the expressions above since it doesn't do anything.